config/config.example.yml

```diff
@@ -208,8 +205,8 @@ https_only: false
 ##   - Software name ("invidious") and version+branch (same data as
 ##     displayed in the footer, e.g: "2021.05.13-75e5b49" / "master")
 ##   - The value of the 'registration_enabled' config (true/false)
-##   - Number of currently connected users
-##   - Average Number of connected users (last month & last 6 months)
+##   - Number of currently registered users
+##   - Number of users who registered in the last month / last 6 months
```

Comment:
It's the number of registered users who connected in the last month/6 months.

Reply:
Heck xD